app/views/devise/passwords/new.haml

```diff
@@ -26,7 +26,7 @@
                        autofocus:      true,
                        aria:           {labelledby: "passwordLabel"}
 
-      = f.submit t('devise.passwords.new.send_password_instructions'), class: "btn btn-block btn-default"
+      = f.submit t('devise.passwords.new.send_password_instructions'), class: "btn btn-block btn-primary"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.